test/integration/bundles.js

```diff
@@ -47,9 +46,27 @@ describe.skip('Bundles - Integrations', () => {
 
     scenario = new ScenarioBuilder(apparatus.identityManager, apparatusScenarioProcessor(apparatus));
     await scenario.addAdminAccount(adminAccountWithSecret);
-
-    const from = getDefaultAddress(await createWeb3());
-    await apparatus.bundleProofRegistryContract.methods.addToWhitelist(from, url).send({from});
+    const from = apparatus.identityManager.nodeAddress();
```

Comment:
Can we please stope using `from` as a name for the sender? I know it makes providing this into the send options easy, but it also makes it really hard to reason about what this is:D